libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -153,9 +153,14 @@ void AP_Camera::trigger_pic()
     case CamTrigType::relay:
         relay_pic();            // basic relay activation
         break;
+#if HAL_SOLO_GIMBAL_ENABLED
     case CamTrigType::gopro:  // gopro in Solo Gimbal
         AP_Camera_SoloGimbal::gopro_shutter_toggle();
         break;
+#else
+    default:
```

Comment:
you should just add this outside the #else

Reply:
Fixed